lib/toolbar-android.js

```diff
@@ -24,6 +28,15 @@ export default function createToolbarAndroidComponent(
   getImageSource
 ) {
   return class IconToolbarAndroid extends PureComponent {
+    constructor(props) {
+      super(props);
+      if (!ToolbarAndroid) {
+        throw new Error(
+          "ToolbarAndroid is not found, Maybe you don't added @react-native-community/toolbar-android in your dependencies"
```

Comment:
`ToolbarAndroid is not found, Maybe you don't added @react-native-community/toolbar-android in your dependencies` to `ToolbarAndroid is not found, Maybe you have not added @react-native-community/toolbar-android in your dependencies` would make more sense I guess?

Reply:
You are right. What is your suggestion?

Reply:
Done.

Thank you.